### DISCUSSION TITLE
Wix burn thirdparty MsiPackages  with ALLUSERS property already set

### DISCUSSION THREAD

**Discussion by woscostametaxa** [wixtoolset/issues#7220] [Questions] [answered]
I have a question with regards of MsiPackage Elements, ALLUSERS property, the usage of Package Installscope (which I can use if I build the msi package) and 3rdparty Msi Packages with that property set already.

Example:
```xml
<?xml version="1.0" encoding="utf-8" standalone="yes"?>

<Wix xmlns:util="http://wixtoolset.org/schemas/v4/wxs/util" xmlns:bal="http://wixtoolset.org/schemas/v4/wxs/bal" xmlns:dep="http://wixtoolset.org/schemas/v4/wxs/dep" xmlns="http://wixtoolset.org/schemas/v4/wxs">
  <Bundle Name="Example)" Manufacturer="manufacturer" Version="1.0.0.0" UpgradeCode="4fb56359-9960-4d31-a24c-a148cce8957d">
    <BootstrapperApplication>
		<bal:WixStandardBootstrapperApplication Theme="none" />
	</BootstrapperApplication>

    <Chain>
      <PackageGroupRef Id="NetFx472RedistAsPrereq"  />
	  <MsiPackage Id="Microsoft_Visual_Studio_2005_x86_Redist"
	   SourceFile="D:\remote_wix4\PackageData\Build\Pipeline\Resources\VCRedist\vc80\vcredist.msi"
	   After="NetFx472Redist"
   	   Cache="remove"
	   Permanent="yes"
	   Vital="yes"
	   Compressed="yes"
	   Visible="yes"
	   >
	   <!-- 	   
	   InstallCondition="INSTALLPREREQUISITES=1"
	   ForcePerMachine="yes"-->
        <Payload SourceFile="D:\remote_wix4\PackageData\Build\Pipeline\Resources\VCRedist\vc80\vcredis1.cab" />
      </MsiPackage>
    </Chain>
  </Bundle>
</Wix>
```
Without "ForcePerMachine="yes" - which has previously worked with Wix 3.11.2 i get the following output:
`wix.exe build -ext WixToolset.UI.wixext -ext WixToolset.Util.wixext -ext WixToolset.Bal.wixext -ext WixToolset.Dependency.wixext -ext WixToolset.NetFx.wixext -culture en-us D:\minimalexample.wxs -out D:\test.exe`

`D:\minimalexample.wxs(11) : warning WIX1133: Bundles require a package to be either per-machine or per-user. The MSI 'Microsoft_Visual_Studio_2005_x86_Redist' ALLUSERS Property is set to '2' which may change from per-user to per-machine at install time. The Bundle will assume the package is per-machine and will not work correctly if that changes. If possible, remove the Property with Id='ALLUSERS' and use Package/@InstallScope attribute instead.`

if I add the ForcePerMachine="yes" to the MsiPackage Attribute list:
`wix.exe build -ext WixToolset.UI.wixext -ext WixToolset.Util.wixext -ext WixToolset.Bal.wixext -ext WixToolset.Dependency.wixext -ext WixToolset.NetFx.wixext -culture en-us D:\minimalexample.wxs -out D:\test.exe`

`D:\minimalexample.wxs(11) : error WIX0365: The 'ALLUSERS' MsiProperty is controlled by the bootstrapper and cannot be authored. (Illegal properties are: 'ACTION', 'ADDLOCAL', 'ADDSOURCE', 'ADDDEFAULT', 'ADVERTISE', 'ALLUSERS', 'REBOOT', 'REINSTALL', 'REINSTALLMODE', or 'REMOVE'.) Remove the MsiProperty element.`

Well - I want to force it per machine, but I dont build the VC 2005 Redist (in my case I need it, and I suppose it happens with other Msi Packages too) - how can I circumvent that error? Can I set the error to a warning?

Is it impossible to build a burn package with a 2005er VC redist? Or any MsiPackage that has ALLUSERS pre-set already? What would be my course of action in this regard?

**Comment by rseanhall** [comment, marked as answer]
Sounds like a bug

**Reply by woscostametaxa** [reply]
Should I create an issue?

**Reply by rseanhall** [reply]
Yes
